### PR TITLE
OCPBUGSM-34079: remove default os/release images files

### PR DESCRIPTION
**Pull request by danielerez**
# Assisted Pull Request

## Description

Removed default os/images json files until properly handle [tools/ocp_version_update](https://github.com/openshift/assisted-service/tree/master/tools/ocp_version_update) tool - [MGMT-7565](https://issues.redhat.com/browse/MGMT-7565)
I.e. the tool is currently updating only default_ocp_versions.json: https://github.com/openshift/assisted-service/pull/2449.
It should update also default_os_images.json and default_release_images.json, otherwise it has no effect as values are overridden by the images files.

## List all the issues related to this PR

- [ ] New Feature
- [x] Bug fix
- [ ] Tests
- [ ] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [ ] Automation (CI, tools, etc)
- [x] Cloud
- [ ] Operator Managed Deployments
- [ ] None

## How was this code tested?

- [x] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [ ] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [ ] No tests needed

## Assignees

/cc @oshercc 
/cc @YuviGold 
/cc @filanov 
/cc @gamli75 

## Checklist

- [x] Title and description added to both, commit and PR.
- [x] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [x] Reviewers have been listed
- [x] This change does not require a documentation update (docstring, `docs`, README, etc)
- [x] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- [ ] Are the title and description (in both PR and commit) meaningful and clear?
- [ ] Is there a bug required (and linked) for this change?
- [ ] Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: openshift/template.yaml [hunk 1]
```diff
@@ -14,10 +14,10 @@ parameters:
...
   required: false
 - name: OS_IMAGES
-  value: '[{"openshift_version":"4.6","cpu_architecture":"x86_64","url":"https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.6/4.6.8/rhcos-4.6.8-x86_64-live.x86_64.iso","rootfs_url":"https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.6/4.6.8/rhcos-live-rootfs.x86_64.img","version":"46.82.202012051820-0"},{"openshift_version":"4.7","cpu_architecture":"x86_64","url":"https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.7/4.7.13/rhcos-4.7.13-x86_64-live.x86_64.iso","rootfs_url":"https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.7/4.7.13/rhcos-live-rootfs.x86_64.img","version":"47.83.202105220305-0"},{"openshift_version":"4.8","cpu_architecture":"x86_64","url":"https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.8/4.8.2/rhcos-4.8.2-x86_64-live.x86_64.iso","rootfs_url":"https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.8/4.8.2/rhcos-live-rootfs.x86_64.img","version":"48.84.202107202156-0"}]' # os images
+  value: '' # os images
```
Comment (gamli75) [inline openshift/template.yaml:17]: why did you remove it?

**Review by gamli75 — COMMENTED** [review]
I don't understand what I should do if I want to update the OCP version on the cloud without updating the service - what exactly do I need to change from now - OPENSHIFT_VERSIONS or OS_IMAGES & RELEASE_IMAGES or all 3 fields ?

File: openshift/template.yaml (hunk 1 above)
Comment (danielerez) [reply openshift/template.yaml:17]: Removed it until handling the ocp_version_update.py tool, just to restore the old behaviour for now.

**Comment by danielerez** [comment]
> I don't understand what I should do if I want to update the OCP version on the cloud without updating the service - what exactly do I need to change from now - OPENSHIFT_VERSIONS or OS_IMAGES & RELEASE_IMAGES or all 3 fields ?

So to update the versions, both OS_IMAGES and RELEASE_IMAGES needs to be updated.
The old OPENSHIFT_VERSIONS still has the values just as a fallback until we could exclusively use these new env vars.
I'll restore the files again after handling the ocp_version_update.py tool to avoid further confusion.

**Comment by gamli75** [comment]
> > I don't understand what I should do if I want to update the OCP version on the cloud without updating the service - what exactly do I need to change from now - OPENSHIFT_VERSIONS or OS_IMAGES & RELEASE_IMAGES or all 3 fields ?
> 
> So to update the versions, both OS_IMAGES and RELEASE_IMAGES need to be updated.
> The old OPENSHIFT_VERSIONS still has the values just as a fallback until we could exclusively use these new env vars.
> I'll restore the files again after handling the ocp_version_update.py tool to avoid further confusion.

Don't forget that we need to update the https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/data/services/assisted-installer/cicd/saas.yaml as well when you'll remove the OPENSHIFT_VERSIONS

**Comment by danielerez** [comment]
> > > I don't understand what I should do if I want to update the OCP version on the cloud without updating the service - what exactly do I need to change from now - OPENSHIFT_VERSIONS or OS_IMAGES & RELEASE_IMAGES or all 3 fields ?
> > 
> > 
> > So to update the versions, both OS_IMAGES and RELEASE_IMAGES need to be updated.
> > The old OPENSHIFT_VERSIONS still has the values just as a fallback until we could exclusively use these new env vars.
> > I'll restore the files again after handling the ocp_version_update.py tool to avoid further confusion.
> 
> Don't forget that we need to update the https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/data/services/assisted-installer/cicd/saas.yaml as well when you'll remove the OPENSHIFT_VERSIONS

Right, it might take a while as we still need 'display_name' for the UI and support_level/default values.
But will update the saas.yaml as required.

**Comment by danielerez** [comment]
Closing, already handled ocp_version_update tool: https://github.com/openshift/assisted-service/pull/2525
